test/Libraries/Microsoft.Extensions.Telemetry.Tests/Logging/Utils.cs

```diff
@@ -24,21 +24,21 @@ public static ILoggerFactory CreateLoggerFactory(Action<ILoggingBuilder>? config
         return new DisposingLoggerFactory(loggerFactory, serviceProvider);
     }
 
-    private sealed class DisposingLoggerFactory : ILoggerFactory
+    internal sealed class DisposingLoggerFactory : ILoggerFactory
     {
         private readonly ILoggerFactory _loggerFactory;
 
-        private readonly ServiceProvider _serviceProvider;
+        internal readonly ServiceProvider ServiceProvider;
```

Comment:
Fields should only be private to the containing class.
```suggestion
        internal ServiceProvider ServiceProvider { get; }
```

Reply:
Done